databases/gcp-postgresql-ssl/README.md

```diff
@@ -0,0 +1,32 @@
+# GCP PostgresQL with SSL
+
+Google Cloud PostgresQL with SSL
+
+Prisma docs: https://www.prisma.io/docs/concepts/database-connectors/postgresql#configuring-an-ssl-connection
+GCP docs: https://cloud.google.com/sql/docs/postgres/configure-ssl-instance
+
+## How to run this locally
+
+### Environment variables
+
+The environment variable `GCP_POSTGRESQL_SSL_DB_URL` should point to a GCP Postgres database. Download the certificates based on GCP docs and put them in the root directory next to this `README.md` file.
+
+The connection string `GCP_POSTGRESQL_SSL_DB_URL` would look like:
+
+`postgresql://<user>:<password>@<host>:<port>/<database>?sslmode=require&sslcert=../server-ca.pem&sslidentity=../client-identity.p12&sslpassword=<client-identity-password>&sslaccept=accept_invalid_certs`
+
+The following files are expected:
+
+- server-ca.pem
+- client-cert.pem
+- client-key.pem
```

Comment:
The environment variable `GCP_POSTGRESQL_SSL_DB_URL` should point to a GCP PostgresQL database. It should follow the following pattern:

`postgresql://<user>:<password>@<host>:<port>/<database>?sslmode=require&sslcert=../server-ca.pem&sslidentity=../client-identity.p12&sslpassword=<client-identity-password>&sslaccept=accept_invalid_certs`

The 3 mentioned files `server-ca.pem`, `client-cert.pem`, `client-key.pem` are to be downloaded from GCP and put into the root directory next to this `README.md` file. (In the tests of this project, these files are created on demand from environment variables that contain their content)